src/PKSim.Presentation/Presenters/Main/StatusBarPresenter.cs

```diff
@@ -154,7 +154,7 @@ public void Handle(ApplicationInitializedEvent eventToHandle)
       {
          updateProjectInfo(Captions.None, Captions.None, false);
          update(StatusBarElements.Version)
-            .WithCaption(_applicationConfiguration.FullVersionDisplay);
+            .WithCaption($"Version {_applicationConfiguration.FullVersionDisplay}");
```

Comment:
This will show the following
![image](https://user-images.githubusercontent.com/1041237/57816868-f00a4180-774a-11e9-8aaf-667565cc5a87.png)

Reply:
please change it back to the full version

Reply:
This is the full version. Version 8 or sthg. 
IF we are in EAP, or beta, it will say so here as well